gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -705,6 +714,15 @@ public JsonWriter newJsonWriter(Writer writer) throws IOException {
   }
 
   /**
+   * Returns a new JSON writer configured for the settings on this Gson instance.
+   */
+  public JsonReader newJsonReader(Reader reader) {
```

Comment:
The method body seems fairly straightforward. Why can't the developer do this themselves?
We could just expose an isLenient() method instead.